src/fidesops/ops/tasks/__init__.py

```diff
@@ -1,13 +1,30 @@
-from typing import Any, Dict, MutableMapping
+from typing import Any, ContextManager, Dict, MutableMapping
 
-from celery import Celery
+from celery import Celery, Task
 from celery.utils.log import get_task_logger
 from fideslib.core.config import load_toml
+from fideslib.db.session import get_db_session
+from sqlalchemy.orm import Session
 
 from fidesops.ops.core.config import config
 
 logger = get_task_logger(__name__)
 
+EMAIL_QUEUE_NAME = "fidesops.email"
+
+
+class DatabaseTask(Task):  # pylint: disable=W0223
+    _session = None
+
+    @property
+    def session(self) -> ContextManager[Session]:
+        """Creates Session once per process"""
+        if self._session is None:
+            SessionLocal = get_db_session(config)
+            self._session = SessionLocal()
+
+        return self._session
```

Comment:
Note to self: this was created for the privacy request runner service to use a single session per process.  It seems to have been moved here to re-use with dispatch_email_task

Reply:
That's good context, thanks for flagging — right now the email sends are reads to the `EmailConfig` and `PrivacyRequest` table, and not writes, so I don't see sessions interfering with one another.

Some constraints I had in mind were:
- it seems superfluous to have multiple application DB connections across all emails for simple read requests
- it's potentially insecure to serialise the secrets in the application server and pass them into the queue with the job (since they'd be stored in Redis for that duration)